incubator/virtualcluster/pkg/syncer/conversion/equality.go

```diff
@@ -25,8 +26,25 @@ import (
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/utils/pointer"
+
+	syncerutil "github.com/kubernetes-sigs/multi-tenancy/incubator/virtualcluster/pkg/syncer/utils"
 )
 
+var StandaloneObjAnnotationPrefixes = []string{"tenancy.x-k8s.io"}
+
+func init() {
+	// load the StandaloneObjAnnotationPrefixex from envs. When the syncer
+	// run in a pod, the env will be set through a configmap
+	// TODO should we add a feature to restart the syncer if the configmap
+	// is updated
+	soap := os.Getenv("STDLN_OBJ_ANNO_PREFXS")
+	if soap == "" {
+		return
+	}
+	// the delimiter of the prefix list is space
+	StandaloneObjAnnotationPrefixes = append(StandaloneObjAnnotationPrefixes, strings.Split(soap, " ")...)
```

Comment:
dedup and remove space-only prefix

may there is a continuous space.

Reply:
use `strings.Field` instead